examples/foc/Kconfig

```diff
@@ -326,6 +326,11 @@ config EXAMPLES_FOC_OPENLOOP_Q
 	default 200
 	depends on EXAMPLES_FOC_HAVE_OPENLOOP
 
+config EXAMPLES_FOC_FEEDFORWARD
+	bool "FOC use feedforward compensation"
+	select INDUSTRY_FOC_FEEDFORWARD
+	default n
+
```

Comment:
Please include a help line to let user know more about it

Reply:
done